yarn.lock

```diff
@@ -831,12 +831,10 @@
   dependencies:
     "@octokit/openapi-types" "^7.2.3"
 
```

Comment:
I am unfamiliar with yarn, so if I messed something up here please let me know. This diff came from:
```
yarn remove @octokit/webhooks
yarn add -D @octokit/webhooks-types@latest
```